### DISCUSSION TITLE
ImportBundle: existing Project assignment not recreated when using the `--global`

### DISCUSSION THREAD

**Discussion by jimlucas** [kimai/kimai#4842] [General]
Initial Issue I created ImportBundle Issue https://github.com/kevinpapst/ImportBundle/issues/16

The problem of duplicates was resolved, but now I have a different issue related to the use of the `--global` option.

After the import all Activities are now "Global".  The previously set Project assignments were NOT recreated during the import.

The description for the `--global` option says "activities without mapping will be created globally".  What is "mapping" referring to?

In the old instance all Activities were assigned to a Project.

Is this the mapping it refers to or is it something different?

**Comment by kevinpapst** [comment]
> In the old instance all Activities were assigned to a Project.

Then don't use `global`.

The new Kimai does not know Activities tat can be mapped to several projects.
